### ISSUE TITLE
U+208B 'SUBSCRIPT MINUS' has wrong width

### ISSUE THREAD

**Issue by Holzhaus** [adobe-fonts/source-code-pro#265]
The subscript minus character does not have the same width as the other characters, which breaks diagrams:

![Screenshot from 2021-02-19 14-32-38](https://user-images.githubusercontent.com/1834516/108511008-8cb62780-72bf-11eb-80be-edf3741b540b.png)

**Comment by frankrolf** [comment]
This character is not supported in SCP, what you are seeing is font fallback (I’m really wondering why code editors would choose to fall back to a different-pitch font?)
In your diagram, I see a few other characters that are not supported by SCP, such as ⦻ (U+29BB) – if that is it.

Please edit your issue to make it a glyph support request, perhaps describing some usage scenario for which other yet unsupported character may be needed. (those diagrams look interesting!)
Thanks!

**Comment by Holzhaus** [comment]
> This character is not supported in SCP, what you are seeing is font fallback (I’m really wondering why code editors would choose to fall back to a different-pitch font?)

Ah ok. To elaborate, the screenshot at the top is actually not a code editor, it's the HTML documentation generated by rustdoc, so it's the browser that messes up here :D

> I see a few other characters that are not supported by SCP, such as ⦻ (U+29BB) – if that is it. Please edit your issue to make it a glyph support request, perhaps describing some usage scenario for which other yet unsupported character may be needed. (those diagrams look interesting!)

I'll edit the description. Thanks!

**Comment by ValZapod** [comment]
> it's the browser that messes up here

Yeah, sorry about that. It is a known bug in chromium: https://bugs.chromium.org/p/chromium/issues/detail?id=807361#c_ts1600152903 

Also see my comment in the other related issue https://bugs.chromium.org/p/chromium/issues/detail?id=1083930#c10 

You should set some other font like PragmataPro.
